MobileCenter/MobileCenter/Internals/MSServiceInternal.h

```diff
@@ -46,4 +46,9 @@
  */
 + (NSString *)logTag;
 
+/**
+ * Service unique key
+ */
+@property(nonatomic, copy, readonly) NSString *serviceName;
```

Comment:
You probably don't need this in this header.
`storageKey` should be removed from here but we will take care of that from develop branch.
Just remove this property.